Eventnet/Eventnet.Api/Services/SaveServices/IPublishEventService.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Eventnet.Services.SaveServices;
+
+public interface IPublishEventService : IDisposable
+{
+    Task SendAsync(string message);
```

Comment:
Ну раз это IPublishService, то пусть функция и называется соответствующе?
```suggestion
    Task PublishAsync(string message);
```